tools/cldr/cldr-to-icu/src/test/java/org/unicode/icu/tool/cldrtoicu/mapper/LocaleMapperTest.java

```diff
@@ -65,7 +65,7 @@ public void testInheritedValuesNotIncludedByDefault() {
             ldml("units/durationUnit[@type=\"foo\"]/durationUnitPattern", "Bar"),
             simpleResult("/durationUnits/foo", "Bar"));
         //ldml/localeDisplayNames/keys/key[@type="(%A)"] ; /Keys/$1
-        addInheritedMapping("xx",
+        addRootMapping(
```

Comment:
I'm a bit unhappy with "root" here, it's only important that it's a parent locale, not the root itself.
Happy to change if you want.